tests-2015/Cargo.toml

```diff
@@ -11,6 +11,7 @@ edition = "2015"
 build = "../tests/src/build.rs"
 
 [lib]
+name = "tests"
 doctest = false
```

Comment:
There is `doctest = false` in both crates, but doctests are compiled by `cargo test` even without the `--doc` option.